tests/security/ima/evm_protection_hmacs.pm

```diff
@@ -50,7 +50,7 @@ sub run {
     assert_script_run "setfattr -x security.ima $sample_app";
     validate_script_output "getfattr -m security.evm -d $sample_app", sub { m/^$/ };
 
-    replace_grub_cmdline_settings('evm=fix ima_appraise=fix', '', 1);
+    replace_grub_cmdline_settings('evm=fix ima_appraise=fix', '', update_grub => 1);
```

Comment:
I'd remove `'',` as it's now not needed :).

Reply:
copy that.

Reply:
@pevik Hi sir, why this ```''```, is not needed,
```
  1087 sub replace_grub_cmdline_settings {
  1088     my $old  = shift;
  1089     my $new  = shift;
  1090     my %args = testapi::compat_args(
  1091         {
  1092             old         => $old,
  1093             new         => $new,
  1094             update_grub => 0,
  1095             search      => get_cmdline_var(),
  1096         }, ['old', 'new', 'update_grub', 'search'], @_);
  1097     change_grub_config($old, $new, $args{search}, "g", $args{update_grub});
  1098 }

if remove it, line 1089, what will do ?
```

Reply:
sorry, Just a little bit confuse this in my brain, ...